tests/integration/defs/stress_test/stress_test.py

```diff
@@ -614,48 +735,113 @@ def run_genai_perf_process(cmd, test_start_time, test_timeout, server_config):
         test_start_time: Start time of the test
         test_timeout: Timeout for the test in seconds
         server_config: Server configuration object
+        request_counter: Optional counter to track requests
 
     Returns:
         Boolean indicating whether the process completed successfully
     """
     # Start genai-perf process with our context manager
-    with launch_process(cmd, start_new_session=True) as process:
+    # Use proper filter pattern for genai-perf
+    http_log_pattern = r'INFO: .+ - "POST /v1/completions HTTP/1.1" 200 OK'
```

Comment:
This is the log of `trtllm-serve` server, why are you matching it in `genai-perf`'s output?

Reply:
yes, it is a mistake